modules/find-policy-violations.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-cluster-configuration.adoc
+:_module-type: PROCEDURE
+[id="find-policy-violations_{context}"]
+= Finding policy violations
+
+[role="_abstract"]
+The *Policy Violations by Severity* widget in the *Configuration Management* view displays policy violations in a sunburst chart.
+Each level of the chart is represented by one ring or circle.
+
+* The innermost circle represents the total number of violations.
+* The next ring represents policy categories (*Low*, *Medium*, *High*, and *Critical*).
+* The outermost ring represents individual policies in a particular category.
+
+The *Configuration Management* view only shows the information about policies that have the *Lifecycle Stage* set to *Deploy*.
+It does not include policies that address runtime behavior or those configured for assessment in the *Build* stage.
+
+
+.Procedure
+. Navigate to the RHACS portal and click *Configuration Management* from the left-hand navigation menu.
+. On the *Secrets Most Used Across Deployments* widget, move your mouse over the sunburst chart to view details about policy violations.
+. Select *_n_ rated as high* (where _n_ is a number) to view detailed information about high-priority policy violations.
+The *Policies* view displays a list of policy violations filtered based on the selected category.
+. Select a row to view more details, including policy description, remediation, deployments with violations, and more.
+The details are visible in a panel.
+. The *Policy Findings* section in the information panel lists deployments where these violations occurred.
+. Select a deployment under the *Policy Findings* section to view related details including Kubernetes labels, annotations, service account, and violation comments and tags.
```

Comment:
You could potentially replace "deployment" with "cluster" to be more explicit, unless deployment is referring to something else. The same applies to other references to deployment in the PR.

Reply:
No its not a cluster, its information about the Kubernetes deployment object. This is correct.